order/batch_verifier_test.go

```diff
@@ -469,8 +503,8 @@ func TestBatchVerifier(t *testing.T) {
 					// bid1ExecFee - bid2ExecFee - chainFees
 					// - bid2SelfChanBalance
 					// 500_000 - 1000 - 1000 -
-					// 1_110 - 1_110 - 186 - 50
-					Value:    495_544,
+					// 1_110 - 1_110 - 186 - 100_000
+					Value:    395_594,
```

Comment:
Did these values need to be changed for the push amount calculation to work correctly? Unclear why we need to increase this and the one above when they aren't using the new feature.